build.sbt

```diff
@@ -44,8 +43,9 @@ lazy val benchmark = project.in(file("serial-benchmark"))
   .settings(moduleName := "finagle-serial-bencrhmark")
```

Comment:
Mind hitting the typo here ("bencrhmark") before merging? Otherwise looks good to me.